### ISSUE TITLE
Terraform v0.12.27 - Crash on version, init, plan, etc...

### ISSUE THREAD

**Issue by trinitronx** [hashicorp/terraform#25386]
### Terraform Version

```
$ terraform version
runtime: netpoll: break fd ready for -2
fatal error: runtime: netpoll: break fd ready for something unexpected

runtime stack:
runtime.throw(0x2a2e35a, 0x39)
	/usr/local/go/src/runtime/panic.go:1116 +0x72
runtime.netpoll(0x1a39a4, 0x1b6547fccf01)
	/usr/local/go/src/runtime/netpoll_kqueue.go:140 +0x30a
runtime.findrunnable(0xc00004e800, 0x0)
	/usr/local/go/src/runtime/proc.go:2323 +0x72b
runtime.schedule()
	/usr/local/go/src/runtime/proc.go:2520 +0x2fc
runtime.park_m(0xc000000c00)
	/usr/local/go/src/runtime/proc.go:2690 +0x9d
runtime.mcall(0x80000)
	/usr/local/go/src/runtime/asm_amd64.s:318 +0x5b

goroutine 1 [runnable, locked to thread]:
runtime.doInit(0x3cb5820)
	/usr/local/go/src/runtime/proc.go:5398 +0xd8
runtime.doInit(0x3cd5fe0)
	/usr/local/go/src/runtime/proc.go:5409 +0x57
runtime.doInit(0x3ccc440)
	/usr/local/go/src/runtime/proc.go:5409 +0x57
runtime.main()
	/usr/local/go/src/runtime/proc.go:190 +0x1ce
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1373 +0x1

goroutine 18 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc00009cf00)
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:154 +0x100
created by go.opencensus.io/stats/view.init.0
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:32 +0x57
```

Note: This crash appears to even affect `terraform version` command.

The version is:  `v0.12.27`
Running on: `macOS Mojave 10.14.6 64-bit`

### Terraform Configuration Files

N/A

### Debug Output

```
export TF_LOG=trace;  terraform version
runtime: netpoll: break fd ready for -2
fatal error: runtime: netpoll: break fd ready for something unexpected

runtime stack:
runtime.throw(0x2a2e35a, 0x39)
	/usr/local/go/src/runtime/panic.go:1116 +0x72
runtime.netpoll(0xabe69, 0x1ba7b6ad7201)
	/usr/local/go/src/runtime/netpoll_kqueue.go:140 +0x30a
runtime.findrunnable(0xc00004c000, 0x0)
	/usr/local/go/src/runtime/proc.go:2323 +0x72b
runtime.schedule()
	/usr/local/go/src/runtime/proc.go:2520 +0x2fc
runtime.park_m(0xc000000c00)
	/usr/local/go/src/runtime/proc.go:2690 +0x9d
runtime.mcall(0x80000)
	/usr/local/go/src/runtime/asm_amd64.s:318 +0x5b

goroutine 1 [runnable, locked to thread]:
path/filepath.Clean(0x7ffeefbff1e0, 0x4f, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:88 +0xc4d
github.com/kardianos/osext.executableClean(0x3cb7360, 0xc00000f0b8, 0x0, 0xe)
	/go/pkg/mod/github.com/kardianos/osext@v0.0.0-20190222173326-2bc1f35cddc0/osext.go:14 +0x3b
github.com/kardianos/osext.init()
	/go/pkg/mod/github.com/kardianos/osext@v0.0.0-20190222173326-2bc1f35cddc0/osext.go:10 +0x22

goroutine 6 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc000108f00)
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:154 +0x100
created by go.opencensus.io/stats/view.init.0
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:32 +0x57

```

### Crash Output

```
terraform version
runtime: netpoll: break fd ready for -2
fatal error: runtime: netpoll: break fd ready for something unexpected

runtime stack:
runtime.throw(0x2a2e35a, 0x39)
	/usr/local/go/src/runtime/panic.go:1116 +0x72
runtime.netpoll(0x253d48cae, 0x1bb0a6c64d01)
	/usr/local/go/src/runtime/netpoll_kqueue.go:140 +0x30a
runtime.findrunnable(0xc000051000, 0x0)
	/usr/local/go/src/runtime/proc.go:2323 +0x72b
runtime.schedule()
	/usr/local/go/src/runtime/proc.go:2520 +0x2fc
runtime.park_m(0xc000001080)
	/usr/local/go/src/runtime/proc.go:2690 +0x9d
runtime.mcall(0x80000)
	/usr/local/go/src/runtime/asm_amd64.s:318 +0x5b

goroutine 1 [runnable, locked to thread]:
github.com/ChrisTrenkamp/goxpath/parser.init()
	/go/pkg/mod/github.com/!chris!trenkamp/goxpath@v0.0.0-20170922090931-c385f95c6022/parser/ast.go:19

goroutine 6 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc000108f00)
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:154 +0x100
created by go.opencensus.io/stats/view.init.0
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:32 +0x57
```

### Expected Behavior

Terraform should have output it's version / or started up properly.

### Actual Behavior

GoLang error:

```
runtime: netpoll: break fd ready for -2
fatal error: runtime: netpoll: break fd ready for something unexpected
```

### Steps to Reproduce

- `terraform version`
OR:
- `terraform init` / `terraform plan`, etc...

### Additional Context

Details about released version:

```
unzip -l ~/Downloads/terraform_0.12.27_darwin_amd64.zip
Archive:  ~/Downloads/terraform_0.12.27_darwin_amd64.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
 64736592  06-24-2020 13:37   terraform
---------                     -------
 64736592                     1 file

unzip -d /tmp/  ~/Downloads/terraform_0.12.27_darwin_amd64.zip
Archive:  ~/Downloads/terraform_0.12.27_darwin_amd64.zip
  inflating: /tmp/terraform

shasum -a 256 /tmp/terraform
9436a5ba5ce9c46dc93eae9305e08d5f30926fd5f7cbf93889e5f807e6523036  /tmp/terraform
```

### References

- Looks similar to [FreeBSD Bug 244931][1] (this is on `packer`... which also uses GoLang)

[1]: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=244931

**Comment by TristanPeers** [comment]
+1

**Comment by claudiobizzotto** [comment]
`v0.12.28` is just out of the oven: https://github.com/hashicorp/terraform/releases/tag/v0.12.28

**Comment by pselle** [comment]
Hello! Yes as @claudiobizzotto notes, we released 0.12.28 to fix this issue -- we built the 0.12 release with the Go version meant for 0.13! 0.12.28 has been released and should resolve this issue. Thank you so much for the report!
